modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-customizing-upgrades-for-kernel-modules_{context}"]
+= Customizing upgrades for kernel modules
+
+Use this procedure to upgrade the kernel module while running maintenance operations on the node, including rebooting the node, if needed. To minimize the impact on the workloads running in the cluster, run the kernel upgrade process sequentially, one node at a time.
+
+[NOTE]
+====
+This procedure requires knowledge of the workload utilizing the kernel module and must be managed by the cluster administrator.
+====
+
+.Prerequisites
+
+* Terminate all user application workloads on the node or move them to another node.
+
+* Unload the currently loaded kernel module.
+
+* Ensure that the user workload (the application running in the cluster that is accessing kernel module) is not running on the node prior to kernel module unloading and that the workload is back running on the node after the new kernel module version has been loaded.
+
+* Ensure that the device plugin managed by KMM on the node is unloaded before the upgrade process and reloaded following the upgrade process.
+
+* Due to Kubernetes limitations in label names, the combined length of `Module` name and namespace must not exceed 39 characters.
+
+.Procedure
+
+.Prerequisites
```

Comment:
It seems weird to me that there are two sets of prerequisites. Also, these prerequisites seems like an entire procedure? Which has me thinking that maybe this instance of the word .Prerequisites should be deleted?